_includes/scenarios/integrate/succeeding-attributes-to-update-their-values.md

```diff
@@ -17,7 +17,7 @@ Hence, we will look at the process of creating, sharing and succeeding an Identi
 ### Creating a RepositoryAttribute
 
 Enmeshed allows you to store data about yourself in the form of IdentityAttributes.
-When [creating an IdentityAttribute]({% link _docs_use-cases/use-case-consumption-create-a-repositoryattribute.md %}), it is not necessary to share it immediately with a peer.
+When [creating an IdentityAttribute]({% link _docs_integrate/create-attribute-for-yourself.md %}), it is not necessary to share it immediately with a peer.
```

Comment:
Maybe link to the subsection "Create an IdentityAttribute for yourself" of the "Create Attribute for yourself" scenario here

Reply:
I considered it briefly, but I don't see a reason to skip the first 3 lines.

Reply:
Also makes sense, no strong opinion